will/requirements/base.txt

```diff
@@ -1,15 +1,16 @@
 APScheduler==2.1.2
 beautifulsoup4==4.6.0
-bottle==0.12.7
-CherryPy==3.6.0
+bottle==0.12.19
+CherryPy==8.9.1
 clint==0.5.1
 dill==0.3.3
 dnspython==1.15.0
 fuzzywuzzy==0.15.1
 Jinja2==2.7.3
-Markdown==2.3.1
+Markdown==3.3.4
 MarkupSafe==0.23
 # Temporary fork of natural, until python 3 support is merged: https://github.com/tehmaze/natural/pull/13
+# Temporary fork of natural, until python 3 support is merged: https://github.com/tehmaze/natural/pull/13
```

Comment:
This looks like a duplicate line?

Reply:
This is and I think it was already there? It looked weird but I didn’t want to touch it until I could test it. We might be able to update natural to the original version now?

Reply:
I checked earlier this week and natural did update it with py3 support https://github.com/tehmaze/natural/pull/11 but there hasn't been a new release since 0.2.0 so our only option is to point to a commit.

Reply:
So do we want to change to that commit or just leave it alone?

Reply:
Let's change to the commit. I dug around for how to test and from this example we can just pass a datetime object in:

http://skoczen.github.io/will/plugins/builtins/#parse-natural-time

As that function depend on the usage of natural:

https://github.com/skoczen/will/blob/master/will/mixins/naturaltime.py#L23

Reply:
Ok natural is pinned to the latest commit. I also had to add it to the --extend-exclude list in flake8 because of the way we're installing it.